bpf/kmesh/workload/sockops_tuple.c

```diff
@@ -138,55 +143,97 @@ static inline void record_ip(__u32 ip)
     __u64 key = ip;
     err = bpf_map_update_elem(&map_of_manager, &key, &value, BPF_NOEXIST);
     if (err)
-        BPF_LOG(ERR, KMESH, "record netcookie failed!, err is %d\n", err);
+        BPF_LOG(ERR, KMESH, "record ip failed!, err is %d\n", err);
 }
 
 static inline void remove_ip(__u32 ip)
 {
     __u64 key = ip;
     int err = bpf_map_delete_elem(&map_of_manager, &key);
     if (err && err != -ENOENT)
-        BPF_LOG(ERR, KMESH, "record netcookie failed!, err is %d\n", err);
+        BPF_LOG(ERR, KMESH, "remove ip failed!, err is %d\n", err);
+}
+
+static inline bool conn_from_sim(struct bpf_sock_ops *skops, __u32 ip, __u32 port)
+{
+    __u32 rev_port = bpf_ntohl(skops->remote_port);
```

Comment:
IMO, in oe_23.03 os scenario, it is better to correct the value of `rev_port` instead `port`